lib/analyzer/apt-analyzer.ts

```diff
@@ -3,12 +3,25 @@ import { AnalyzerPkg } from "./types";
 
 export { analyze };
 
-async function analyze(targetImage: string, options?: DockerOptions) {
-  const docker = new Docker(targetImage, options);
-  const dpkgFile = (await docker.catSafe("/var/lib/dpkg/status")).stdout;
+const APT_DPKG_STATUS = "/var/lib/dpkg/status";
+const APT_EXT_STATES = "/var/lib/apt/extended_states";
+
+const APT_PKGFILES = [APT_DPKG_STATUS, APT_EXT_STATES];
+
+export { APT_PKGFILES };
+
+async function analyze(
+  targetImage: string,
+  options?: DockerOptions,
+  files?: {} | void,
+) {
+  if (!files) {
+    files = await new Docker(targetImage, options).extract(APT_PKGFILES);
```

Comment:
I'm a little confused... are these not passed in by the `analyzer/index`?

Reply:
I'll remove this once I'll fix and verify that all tests pass

Reply:
I don't follow... Why not remove now? If you're going to remove it anyway I don't understand why it's needed during review. Am I missing something here?

Reply:
Done